### DISCUSSION TITLE
Collapsible items in multilevel sidebar

### DISCUSSION THREAD

**Discussion by qido2** [vuejs/vitepress#1511] [Q&A] [answered]
Hi, I've noticed some inconsistent behaviour when using a multi level sidebar added in version 1.05.

Ideally I'd like to have the children on second and third level hidden/collapsed until you open the parent item for each one.

I've been trying to fiddle with collapsible and collapsed parameters, and sometimes the children items got hidden and shown when clicking items in completely different group, but this was not consistent across all unassociated groups.

Would it be possible to add collapse button to lower level items?

**Comment by leosin** [comment]
same question
as follows, the level 2 `collapsible` and `collapsed` not working...

```js {3}
{
    text: 'Level 1',
    link: 'links',
    collapsible: true, // working
    collapsed: true, // working
    items: [
      {
        text: 'Level 2',
        link: 'links',
        collapsible: true, // not working
        collapsed: true, // not working
        items: [
          { text: 'text', link: 'link' },
        ],
      },
    ],
  },
```

**Comment by Zhengqbbb** [comment, marked as answer]
ref: https://github.com/vuejs/vitepress/pull/1361 https://github.com/vuejs/vitepress/issues/1360
I sure the feature will coming soon 🤗
